lib/galaxy/job_execution/actions/post.py

```diff
@@ -55,7 +55,9 @@ def execute(cls, app, sa_session, action, job, replacement_dict, final_job_state
         try:
             frm = app.config.email_from
             history_id_encoded = app.security.encode_id(job.history_id)
+            workflow_id_encoded = app.security.encode_id(job.workflow_invocation_step.workflow_invocation_id)
             link = f"{app.config.galaxy_infrastructure_url}/histories/view?id={history_id_encoded}"
+            link_invocation = f"{app.config.galaxy_infrastructure_url}/workflows/invocations/report?id={workflow_id_encoded}"
```

Comment:
```suggestion
            link_invocation = f"{app.config.galaxy_infrastructure_url}/workflows/invocations/report?id={invocation_id_encoded}"
```

(maybe update these both to `invocation_id_encoded` just for clarity since it's not the workflow)